query/src/main/scala/filodb/query/exec/SelectRawPartitionsExec.scala

```diff
@@ -44,18 +60,18 @@ object SelectRawPartitionsExec extends  {
       qLogger.debug(s"Replacing range function $origFunc with $newFunc...")
       replaceRangeFunction(transformers, origFunc, newFunc)
     } else {
-      transformers.toBuffer   // Produce an immutable copy, so the original one is not mutated by accident
+      transformers.toBuffer // Produce an immutable copy, so the original one is not mutated by accident
     }
   }
 
-  def newXFormersForHistMax(schema: Schema,
-                            colIDs: Seq[Types.ColumnId],
-                            transformers: Seq[RangeVectorTransformer]): Seq[RangeVectorTransformer] = {
-    histMaxColumn(schema, colIDs).map { maxColID =>
+  def newXFormersForHistMaxMin(schema: Schema,
+                               colIDs: Seq[Types.ColumnId],
+                               transformers: Seq[RangeVectorTransformer]): Seq[RangeVectorTransformer] = {
+    histMaxMinColumns(schema, colIDs).map { colId =>
```

Comment:
can this be replaced with `histMaxColumn`, dont think col_id is used any where?

Reply:
That is a good point. I added it to make sure the underlying schema must have both max and min ( since the transformers are changed to have both )